ai/ai_model_building/army_builder.py

```diff
@@ -0,0 +1,32 @@
+from unit_parser import UNIT_LIST
+
+
+def unit_binder(unit_name):
+    for unit in UNIT_LIST:
+        if unit['name'] == unit_name:
+            return unit
+    return None
+
+
+def army_builder(player_resume):
+    modifiers = []
+    units = []
+    score = 0
+    score = player_resume['score']
+    modifiers = [hash(modifier) for modifier in player_resume['modifier']]
```

Comment:
What is hash() and why do you use it ?

Reply:
that's why I told I needed to make a call with the rest of the math team to explains what I done
basically as the function name says. I create a hash for each modifier.
A modifier is not just a stat booster, it has some effects. As it's not possible to represent those effects at all, I decided to replace the modifiers by hash.
In the neuronal network, they will act as element that influence the result in a way and I think it's the easier and the more efficient way to represent them